changes/2035.doc.md

```diff
@@ -0,0 +1 @@
+docs: remove docker compose top-level element 'version'
```

Comment:
I think change logs is not a commit message so prefixes like `docs:` are not required. Might be changed something like:
> Remove deprecated `version` from the docker complse YAML templates in package installation docs.

Reply:
changed it.